### PR TITLE
GitHub 71

### PR DESCRIPTION
**Pull request by philbrookes**

### CODE REVIEW

File: pkg/reconciler/service/service.go [hunk 1]
```diff
@@ -2,211 +2,9 @@ package service
 
 import (
```
Comment (astefanutti) [inline pkg/reconciler/service/service.go:3]: Could this file be completely removed?

File: pkg/reconciler/deployment/deployment.go [hunk 2]
```diff
@@ -2,219 +2,9 @@ package deployment
 
 import (
 	"context"
-	"fmt"
```
Comment (astefanutti) [inline pkg/reconciler/deployment/deployment.go:5]: Could this file be completely removed?

File: pkg/reconciler/deployment/controller.go [hunk 3]
```diff
@@ -5,10 +5,8 @@ import (
 	"time"
```
Comment (astefanutti) [inline pkg/reconciler/deployment/controller.go:5]: Could this file be completely removed?

**Review by astefanutti — APPROVED** [review]

**Comment by philbrookes** [comment]
@astefanutti I believe @maleck13 wanted to keep the deployment controller in place but empty as we will want to do work in that area in the future, and this stops us having to wire up the controller all over again.

**Comment by astefanutti** [comment]
@philbrookes I see, that sounds good 👍🏼.

**Pull request merged**